codereview.java

```diff
@@ -0,0 +1,37 @@
+public class RuntimeIssuesDemo {
+
+    public static void main(String[] args) {
+        // 1. ArrayIndexOutOfBoundsException
+        int[] numbers = {1, 2, 3};
+        System.out.println(numbers[3]); // Accessing index 3, which is out of bounds
+
+        // 2. NullPointerException
+        String str = null;
+        System.out.println(str.length()); // Attempting to invoke method on null reference
+
+        // 3. ArithmeticException
+        int result = 10 / 0; // Division by zero
+
+        // 4. NumberFormatException
+        String numStr = "abc";
+        int num = Integer.parseInt(numStr); // Parsing a non-numeric string
```

Comment:
<b>code: int num = Integer.parseInt(numStr); // Parsing a non-numeric string - </b>
Add error handling for NumberFormatException when parsing the string to an integer